packages/frontapp/src/app/components/BookInfo/index.tsx

```diff
@@ -341,15 +360,18 @@ const BookInfo: FC<IBookInfoProps> = ({
   };
 
   const deleteItem = async () => {
-    console.log('deleted');
     await removeMaterial({
       variables: {
         identifier: identifier,
         type: type,
         location_id: location_id,
       },
     });
+    navigate('/search');
+    window.location.reload();
```

Comment:
a refresh is needed to be able to update the data in the backend, and navigate from the page to /search because the url of the current page is item's ID, which will not be available after the book is deleted